plugins/aks.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: aks
+spec:
+  version: v0.2.0
+  homepage: https://github.com/Azure/kubectl-aks
+  shortDescription: Interact with and debug AKS clusters even in challenging situations
```

Comment:
Just fyi this will get truncated a lot. I believe 50 chars is the cutoff.

Reply:
Oh, I thought it was 50. I'll change it to just `Interact with and debug AKS clusters` in another PR.